pkg/plugin/connector/acceptance_testing.go

```diff
@@ -319,68 +264,74 @@ func testSource_Read_Success(t *testing.T, tdf testDispenserFunc) {
 	source, err := dispenser.DispenseSource()
 	is.NoErr(err)
 
-	err = source.Start(ctx, nil)
+	stream := source.NewStream()
```

Comment:
All tests (and generally all code) that were using `Start` were changed as they need to create the stream and use it directly. We have less abstraction now.